### PR TITLE
chore: add save icon to artwork grids

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [CX-3188] 

### Description

This PR adds the save icon to artwork grids

Group 1 1

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Add save icon to artworks grid - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-3188]: https://artsyproduct.atlassian.net/browse/CX-3188?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Components/ArtworkGrids/ArtworkGridItem.tests.tsx [hunk 1]
```diff
@@ -243,19 +243,12 @@ describe("save artworks", () => {
...
 
-  it("is hidden for lots", () => {
-    const saleArtwork = {
-      lotLabel: "Lot 1",
-      sale: {
-        isClosed: false,
-        cascadingEndTimeIntervalMinutes: 1,
-      },
-    }
+  it("is not possible when hideSaveIcon prop is specified", () => {
```
Comment (MrSltun) [inline src/app/Components/ArtworkGrids/ArtworkGridItem.tests.tsx:246]: 👀 
```suggestion
  it("is not visible when hideSaveIcon prop is specified", () => {
```

File: src/app/Components/ArtworkGrids/ArtworkGridItem.tsx [hunk 2]
```diff
@@ -205,8 +205,6 @@ export const Artwork: React.FC<ArtworkProps> = ({
   const canShowAuctionProgressBar =
     !!artwork.sale?.extendedBiddingPeriodMinutes && !!artwork.sale?.extendedBiddingIntervalMinutes
 
-  const isBiddableLot = artwork?.sale?.isAuction && !artwork.sale.isClosed
```
Comment (MrSltun) [inline src/app/Components/ArtworkGrids/ArtworkGridItem.tsx:208]: **Question:** Why are we removing this condition? 👀

**Review by MrSltun — COMMENTED** [review]
Looks great! I just added a couple comments :)

File: src/app/Components/ArtworkGrids/ArtworkGridItem.tsx (hunk 2 above)
Comment (MounirDhahri) [reply src/app/Components/ArtworkGrids/ArtworkGridItem.tsx:208]: first requirement from Pierluca was to only show the save icon for artworks that are not lots, then he changed his mind after speaking to FX and decided that we will show the save icon to all artworks - even lots.

**Review by MrSltun — APPROVED** [review]

**Review by dariakoko — COMMENTED** [review]
⭐ x5

**Pull request merged**